### PR TITLE
improve and add comments

### PR DESCRIPTION
**Pull request by ChungHsuanChen**
tcpclient.c udpserver.c and udpclient.c

## 拉取/合并请求描述：(PR description)

[
Improve and add comments in example/networks folder including tcpserver.c udpserver.c tcpclient.c and udpclient.c.
Comments in those files are added with English .
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [√] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [√] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [√] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [√] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [√] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [√] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [√] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [√] 本拉取/合并使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](../documentation/coding_style_cn.md) This PR complies with [RT-Thread code specification](../documentation/coding_style_en.txt)

### CODE REVIEW

File: examples/network/udpclient.c [hunk 1]
```diff
@@ -27,29 +28,33 @@ static int is_running = 0;
...
+/**
+* @brief  This function is for creating a udp client on RT-Thread
+*/
 static void udpclient(void *arg)
 {
     int sock;
     struct hostent *host;
     struct sockaddr_in server_addr;
-
+    /* Get host address by parameter url(Domain name resolution if input domain) */
```
Comment (mysterywolf) [inline examples/network/udpclient.c:41]: url要大写，括号要空一格。
```suggestion
    /* Get host address by parameter URL (domain name resolution if input domain) */
```

File: examples/network/tcpserver.c [hunk 2]
```diff
@@ -34,46 +35,49 @@
 static int started = 0;
 static int is_running = 0;
 static int port = 5000;
-static const char send_data[] = "This is TCP Server from RT-Thread."; /* 发送用到的数据 */
+static const char send_data[] = "This is TCP Server from RT-Thread.";/* The message be sent */ /* 发送用到的数据 */
```
Comment (mysterywolf) [inline examples/network/tcpserver.c:38]:
```suggestion
static const char send_data[] = "This is TCP Server from RT-Thread."; /* The message be sent */ /* 发送用到的数据 */
```

File: examples/network/tcpclient.c [hunk 3]
```diff
@@ -48,40 +52,42 @@ static void tcpclient(void *arg)
...
     }
-
+    /* Initialize server side address */
     /* 初始化预连接的服务端地址 */
     server_addr.sin_family = AF_INET;
     server_addr.sin_port = htons(port);
     server_addr.sin_addr = *((struct in_addr *)host->h_addr);
     rt_memset(&(server_addr.sin_zero), 0, sizeof(server_addr.sin_zero));
-
+    /*Connect to server*/
```
Comment (mysterywolf) [inline examples/network/tcpclient.c:86]:
```suggestion
    /* Connect to server */
```

File: examples/network/tcpclient.c [hunk 4]
```diff
@@ -35,8 +36,11 @@ static int started = 0;
 static int is_running = 0;
 static char url[256];
 static int port = 8080;
-static const char send_data[] = "This is TCP Client from RT-Thread."; /* 发送用到的数据 */
+static const char send_data[] = "This is TCP Client from RT-Thread.";/* The message be sent */ /* 发送用到的数据 */
```
Comment (mysterywolf) [inline examples/network/tcpclient.c:39]:
```suggestion
static const char send_data[] = "This is TCP Client from RT-Thread."; /* The message be sent */ /* 发送用到的数据 */
```

**Review by mysterywolf — CHANGES_REQUESTED** [review]
感谢贡献pr，请签署CLA

**Review by mysterywolf — APPROVED** [review]
@Guozhanxin 合并建议squash压缩一下，保留中英文我觉得没啥问题 +1

**Comment by ChungHsuanChen** [comment]
> @Guozhanxin 合并建议squash压缩一下，保留中英文我觉得没啥问题 +1

已經squash了喔!謝謝

**Comment by ChungHsuanChen** [comment]
您好，因為這是我first contribution，所以我這邊顯示First-time contributors need a maintainer to approve running workflows，是不是請您approve呢?謝謝

**Comment by ChungHsuanChen** [comment]
您好，不好意思之前格式好像qorkflow不會過我調整後也push上去了，再麻煩您幫忙approve，謝謝

**Comment by mysterywolf** [comment]
感谢 一会会approve 我没这个权限

**Pull request merged**